assets/chezmoi.io/docs/quick-start.md

```diff
@@ -82,7 +82,7 @@ git push -u origin main
 !!! hint
 
     chezmoi can be configured to automatically add, commit, and push changes to
-    your repo.
+    your repo. [read more](https://www.chezmoi.io/user-guide/daily-operations/#automatically-commit-and-push-changes-to-your-repo)
```

Comment:
There are two problems here:

1. We  avoid `read more` links in chezmoi documentation, but instead try to link appropriate words in the context sentence. I would change this to link `[add, commit, and push]`.
2. Unless referring to "the website", we don't link directly to `www.chezmoi.io` but instead use relative links. In this case, the link should be `daily-operations/index.md#automatically-commit-and-push-changes-to-your-repo`…I think. I’m working on a PR which includes a plugin to make cross-links easier to reason about, but it hasn't been completed.